MediaBrowser.MediaEncoding/Attachments/AttachmentExtractor.cs

```diff
@@ -174,19 +211,24 @@ public async Task<(MediaAttachment Attachment, Stream Stream)> GetAttachment(Bas
 
             if (exitCode != 0)
             {
-                failed = true;
-
-                _logger.LogWarning("Deleting extracted attachments {Path} due to failure: {ExitCode}", outputPath, exitCode);
-                try
+                if (isExternal && exitCode == 1)
                 {
-                    if (Directory.Exists(outputPath))
```

Comment:
Why remove this? Directory.Delete will throw if the path doesn't exist.

Reply:
The directory gets created at the beginning and never deleted. The only case in which it wouldn't exist here is if someone manually deleted it at exactly the right second (the whole thing normally takes less than a second, maybe a couple of seconds if you're running from an extremely slow cloud storage or hard drive and have unreasonably large files attached).

This was only in the code to begin with because I took it from the code for extracting single attachment files, where it would be unsure if ffmpeg created the file.

I'll fix the other issues you wrote.